## Covid-19

### Structural biology

https://en.wikipedia.org/wiki/Severe_acute_respiratory_syndrome_coronavirus_2#cite_note-CoronavirusStudyGroup-1

Each SARS-CoV-2 virion is approximately 50–200 nanometres in diameter.[62] Like other coronaviruses, SARS-CoV-2 has four structural proteins, known as the S (spike), E (envelope), M (membrane), and N (nucleocapsid) proteins; the N protein holds the RNA genome, and the S, E, and M proteins together create the viral envelope.


## Mutation Rates

https://en.wikipedia.org/wiki/RNA_virus#Mutation_rates

the RNA  nucleic acid is usually single-stranded RNA (ssRNA) but may be double-stranded RNA (dsRNA).[2] Notable human diseases caused by RNA viruses include Ebola virus disease, SARS, COVID-19.

RNA viruses generally have very high mutation rates compared to DNA viruses,because viral RNA polymerases lack the proofreading ability of DNA polymerases.

## RdRP

RNA-dependent RNA polymerase (RdRP, RDR) or RNA replicase is an enzyme that catalyzes the replication of RNA from an RNA template. This is in contrast to a typical DNA-dependent RNA polymerase, which catalyzes the transcription of RNA from a DNA template.

- Continuous and Discontinuous RNA Synthesis in Coronaviruses,Coronavirus Replication and transcription- https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6025776/



## Coronavirus Replication and Transcription

Coronaviruses are enveloped, positive-strand RNA viruses with genomes approximately 30 kb in length that belong to the family Coronaviridae in the order Nidovirales. 

From the abstract -

> " Coronavirus RNA synthesis is performed by a replication-transcription complex that includes viral and cell proteins that recognize cis-acting RNA elements mainly located in the highly structured 5′ and 3′ untranslated regions. In addition to many viral nonstructural proteins, the presence of cell nuclear proteins and the viral nucleocapsid protein increases virus amplification efficacy. Coronavirus RNA synthesis is connected with the formation of double-membrane vesicles and convoluted membranes. Coronaviruses encode proofreading machinery, unique in the RNA virus world, to ensure the maintenance of their large genome size."

The 5′-proximal two-thirds of the coronavirus genome encodes the replicase gene, which contains two open reading frames, 

**ORF1a and ORF1b (Figure 1a). Translation of ORF1a yields polyprotein 1a (pp1a), and −1 ribosomal frameshifting allows translation of ORF1b to yield pp1ab (5, 6). Together, these polyproteins are co- and posttranslationally processed into 16 nonstructural proteins (nsps), most of them driving viral genome replication and subgenomic mRNA (sgmRNA) synthesis (Figure 1a)**. 

The 3′ third of the genome encodes the structural and accessory proteins, which vary in number among the different coronaviruses.

- Orf- Open Reading Frame https://en.wikipedia.org/wiki/Open_reading_frame. An ORF is a continuous stretch of codons 


- Overview of Reverse Transcription, https://www.ncbi.nlm.nih.gov/books/NBK19424/

- 5', Five Prime Untranslated Region- https://en.wikipedia.org/wiki/Five_prime_untranslated_region

- Internal ribosome entry sites and viruses- https://en.wikipedia.org/wiki/Five_prime_untranslated_region#Other_mechanisms


- Molecular immune pathogenesis and diagnosis of COVID-19, https://www.sciencedirect.com/science/article/pii/S2095177920302045

- Reverse Transcriptase - https://en.wikipedia.org/wiki/Reverse_transcriptase


- Talk about Protein Folding:- 
Protein folding is the physical process by which a protein chain acquires its native 3-dimensional structure, a conformation that is usually biologically functional, in an expeditious and reproducible manner. It is the physical process by which a polypeptide folds into its characteristic and functional three-dimensional structure from a random coil. 


- Severe acute respiratory syndrome coronavirus 2 isolate Wuhan-Hu-1, complete genome,NCBI Reference Sequence: NC_045512.2
https://www.ncbi.nlm.nih.gov/nuccore/NC_045512.2

- https://www.ncbi.nlm.nih.gov/nuccore/MN908947
- https://www.ncbi.nlm.nih.gov/Structure/icn3d/full.html


-Computational_Prediction_of_Mutational_Effects_on_the_SARS-CoV-2_Binding_by_Relative_Free_Energy_- Look for Forcefield and water files https://chemrxiv.org/articles/Computational_Prediction_of_Mutational_Effects_on_the_SARS-CoV-2_Binding_by_Relative_Free_Energy_Calculations/11902623/files/21865008.pdf
(PDB ID: 2AJF) The Amber force field ff14SB23 and the TIP3P water model2
- GROMACS Simulation



## Tests (how they work)
- All based on https://en.wikipedia.org/wiki/Reverse_transcription_polymerase_chain_reaction
- USA -- https://www.fda.gov/media/134922/download
- selected from regions of the virus nucleocapsid (N) gene
    28286---28308--28332---28358
    29163---29187--29210---29230
- https://biosearchtech.a.bigcontent.io/v1/static/coa_KIT-NCOV-PP1-1000_Lot-No-143503
- South Korea -- http://www.kogene.co.kr/eng/about_us/news/listbody.php?h_gcode=board&h_code=7&po_no=288
- E gene detection (same for all coronavirus)
- specific RdRp detection

- Rapid Detection of SARS-CoV-2 Using Reverse transcription RT-LAMP method - https://www.medrxiv.org/content/10.1101/2020.03.02.20030130v2.full.pdf



When you talk about the code. Mention villin headpiece. pdb file:- 1uw7

Deconstructing the Coronavirus


